UnitTests/Text/CollectionNavigatorTests.cs

```diff
@@ -376,5 +376,33 @@ public void MinimizeMovement_True_ShouldStayOnCurrentIfMultipleMatches ()
 
 			Assert.Equal (-1, current = n.GetNextMatchingItem (current, "x", true));
 		}
+
+		[Fact]
+		public void IsCompatibleKey_Does_Not_Allows_Alt_And_Ctrl_Keys ()
```

Comment:
IsCompatibleKey_Does_Not_Allows_Alt_And_Ctrl_Keys -> IsCompatibleKey_Does_Not_Allow_Alt_And_Ctrl_Keys

(I know, I know, English is weird).